### PR TITLE
feat(kube-reset): worker nodes policy control addition

### PR DESCRIPTION
**Pull request by ganeshkumar1989**
Close MBP-245

### Requirements

An option to reinstall worker nodes is required while resetting a kube.

### Description of the Change

The Worker Nodes Policy Control has been added to enable the users to preserve the nodes during the reset.

### CODE REVIEW

File: client/app/kubernetes/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -26,10 +27,11 @@
   "kube_service_billing_contact": "Nic Billing",
   "kube_service_billing_renewal": "Renouvellement",
   "kube_service_reset_title": "Réinitialiser la configuration de votre cluster Kubernetes",
-  "kube_service_reset_message": "En cliquant ici vous remettrez votre service Kubernetes dans son état original. Cela supprimera l'intégralité de vos nœuds de calcul, vos services, leur configuration etc... <strong>Cette action est irréversible</strong>.",
+  "kube_service_reset_message": "En cliquant ici vous remettrez votre service Kubernetes dans son état original. Cela supprimera l'intégralité de vos vos services, leur configuration etc... <strong>Cette action est irréversible</strong>.",
```
Comment (antleblanc) [inline client/app/kubernetes/translations/Messages_fr_FR.json]:
```suggestion
  "kube_service_reset_message": "En cliquant ici vous remettrez votre service Kubernetes dans son état original. Cela supprimera l'intégralité de vos services, leur configuration etc… <strong>Cette action est irréversible</strong>.",
```

Comment (ganeshkumar1989) [reply client/app/kubernetes/translations/Messages_fr_FR.json]: This change is done

File: client/app/kubernetes/nodes/kubernetes-nodes.controller.js [hunk 2]
```diff
@@ -32,12 +32,27 @@ angular.module('managerApp').controller('KubernetesNodesCtrl', class KubernetesN
...
+    ).finally(() => { this.loading = false; });
+  }
+
   getNodes() {
     this.loading = true;
     return this.Kubernetes.getNodes(this.serviceName)
       .then((nodes) => { this.nodes = nodes; })
-      .catch(() => this.CloudMessage.error(this.$translate.instant('kube_nodes_error')))
-      .finally(() => { this.loading = false; });
+      .catch(() => this.CloudMessage.error(this.$translate.instant('kube_nodes_error')));
```
Comment (FredericEspiau) [inline client/app/kubernetes/nodes/kubernetes-nodes.controller.js]: I don't think you want to return the result of `CloudMessage.error`

File: client/app/kubernetes/nodes/kubernetes-nodes.html [hunk 3]
```diff
@@ -31,7 +31,7 @@
                       data-translate="{{:: 'kube_service_cluster_status_' + $row.status }}">
                 </span>
             </oui-column>
-            <oui-action-menu data-disabled="$row.status === 'INSTALLING' || $row.status === 'DELETING'"  data-align="end" data-compact>
+            <oui-action-menu data-disabled="$row.status === 'INSTALLING' || $row.status === 'DELETING' || $ctrl.cluster.status === 'RESETTING'" data-align="end" data-compact>
```
Comment (FredericEspiau) [inline client/app/kubernetes/nodes/kubernetes-nodes.html]: Duplicated code with https://github.com/ovh-ux/ovh-manager-cloud/pull/1268/files#diff-2b1843a9c04498d4d8156544bb90e834R31

This should be refactored in a function :)

**Pull request merged**